test/functional/cmdLineTests/jitserver/playlist.xml

```diff
@@ -27,6 +27,11 @@
 		<variations>
 			<variation>NoOptions</variation>
 		</variations>
+			<disable>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/18599</comment>
+				<version>8</version>
+				<platform>x86-64_linux</platform>
```

Comment:
Do I need to add `.*` to this?

Reply:
No, it is not necessary to have `.*` in this case. But we do need to wrap this with `<disables>...</disables>`.

Reply:
Updated.